### PR TITLE
Ability to change the material for crafting (ITEM FRAME)

### PR DESCRIPTION
**Pull request by Timoti11**

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -23,8 +23,8 @@ dependencies {
 }
 
 group = 'com.spiritothawk.plugins'
-version = '2.2.2'
-description = 'CraftableInvFrames'
+version = '2.2.2.2'
```
Comment (SpiritOTHawk) [inline build.gradle:26]: 2.3.0

File: build.gradle [hunk 2]
```diff
@@ -23,8 +23,8 @@ dependencies {
...
+description = 'CraftableInvFramesPlus'
```
Comment (SpiritOTHawk) [inline build.gradle:27]: Why plus?

File: readme.md [hunk 3]
```diff
@@ -3,8 +3,8 @@
 
 This plugin enables the use of 1.20's invisible item frames for survival players
 
-Invisible item frames are crafted similar to tipped arrows - one lingering invisibility potion surrounded by 8 item frames\
```
Comment (SpiritOTHawk) [inline readme.md:6]: Dont change default recipe

File: src/main/resources/plugin.yml [hunk 4]
```diff
@@ -1,8 +1,8 @@
 name: craftableinvframes
-version: ${project.version}
+version: 2.2.2.2
```
Comment (SpiritOTHawk) [inline src/main/resources/plugin.yml:2]: ${project.version}

**Review by SpiritOTHawk — CHANGES_REQUESTED** [review]

File: build.gradle (hunk 2 above)
Comment (Timoti11) [reply build.gradle:27]: > Why plus?

Little bit more options

File: src/main/resources/plugin.yml (hunk 4 above)
Comment (Timoti11) [reply src/main/resources/plugin.yml:2]: Yes, but i didn't use actions, and I wanna check my version

**Comment by Timoti11** [comment]
And one more... I was not able to implement the tab complete for around and center because this is my first java code. We can do it in our own way. I made this version individually for myself, so not everything is perfect.

**Pull request closed without merging**